docs/usage.md

```diff
@@ -82,78 +134,80 @@ By default, the pipeline expects paired-end data. If you have single-end data, y
 
 It is not possible to run a mixture of single-end and paired-end files in one run.
 
+## Optional arguments
 
-## Reference Genomes
+### Trimming options:
 
-The pipeline config files come bundled with paths to the illumina iGenomes reference index files. If running with docker or AWS, the configuration is set up to use the [AWS-iGenomes](https://ewels.github.io/AWS-iGenomes/) resource.
+    --adapter_forward             Sequence of 3' adapter to remove in the forward reads
+    --adapter_reverse             Sequence of 3' adapter to remove in the reverse reads
+    --mean_quality                Mean qualified quality value for keeping read (default: 15)
+    --trimming_quality            Trimming quality value for the sliding window (default: 15)
 
-### `--genome` (using iGenomes)
-There are 31 different species supported in the iGenomes references. To run the pipeline, you must specify which to use with the `--genome` flag.
+### Binning options:
 
-You can find the keys to specify the genomes in the [iGenomes config file](../conf/igenomes.config). Common genomes that are supported are:
+    --refinem                     Enable bin refinement with refinem.
```

Comment:
These are the ones that ned more explanation really. What does the one-liner mean? When would you consider changing them? What things should be looked out for when doing so?